client/sites-dashboard/components/sites-container.tsx

```diff
@@ -0,0 +1,139 @@
+import { ClassNames } from '@emotion/react';
+import styled from '@emotion/styled';
+import { createInterpolateElement } from '@wordpress/element';
+import { useI18n } from '@wordpress/react-i18n';
+import EmptyContent from 'calypso/components/empty-content';
```

Comment:
Should we use another component and avoid importing one from Calypso?

Reply:
I'm not sure this is an issue 😕 ? I see other places we import from calypso c`lient/sites-dashboard/components/sites-table-row.tsx`

Reply:
There was a discussion that ended up with the conclusion that we shouldn't have dependencies on Calypso. Trying to clarify it on p1657261784919289-slack-C0347E545HR

Reply:
Thank you for the discussion thread. I'm wondering though since these components exist it feels kinda weird recreating them. Maybe we should move them to `packages/` but that might not be in the scope of this work in terms of impact it might have if we move them.

Reply:
To avoid blocking this pull request, I think we might want to move this to a follow-up issue.

Reply:
Yeah I agree